corehq/apps/userreports/tasks.py

```diff
@@ -208,77 +200,6 @@ def _iteratively_build_table(config, resume_helper=None, in_place=False, limit=-
             current_config.save()
 
 
-@task(serializer='pickle', queue=UCR_CELERY_QUEUE)
-def compare_ucr_dbs(domain, report_config_id, filter_values, sort_column=None, sort_order=None, params=None):
-    if report_config_id not in settings.UCR_COMPARISONS:
```

Comment:
`settings.UCR_COMPARISONS` can be removed

Reply:
0d1f65a167ec49d37e686487bf6e34d9f4e6a68f